packages/ui/common/src/lib/components/json-view/json-view.component.scss

```diff
@@ -16,3 +16,8 @@
 .code-header{
   padding: 0.625rem 1rem;
 }
+
+.wrap-text {
+  white-space: pre-wrap;
+  word-wrap: normal;
+}
```

Comment:
Hi there, could you please use tailwind in the html template instead of this? "ap-whitespace-nowrap ap-break-normal "